README.md

```diff
@@ -1868,6 +1868,7 @@ Update Time, five active automations, webhooks.
 
 ## Remote Desktop Tools
 
+  * [ZAssist](https://www.zoho.com/assist/free-remote-access.html) — Zoho Assist's forever free plan includes one technician and 5 URS computer licenses, enabling you to manage your assets remotely.
```

Comment:
Please add it to the existing entry for zoho - we only list companies once

Reply:
Yeah, but if we put this service under the other Zoho service, it might not click for the user. The right spot for it is under the remote control tool.

Reply:
I agree it has some benefits, we still only list them once. So please move there.

Reply:
Okay, the change has been made. Please check now.